examples/GPIO/Blink/Blink.ino

```diff
@@ -0,0 +1,40 @@
+/*  Example of reading and writing USB Host Shield GPI output
+    This example uses "Wiring" style interface. See Blink_LowLevel for example of using low-level UHS interface
+    Author: Brian Walton (brian@riban.co.uk)
+*/
+#include <UHS2_gpio.h>
+
+// Satisfy the IDE, which needs to see the include statment in the ino too.
+#ifdef dobogusinclude
+#include <spi4teensy3.h>
+#endif
+
+#define OUTPUT_PIN 0
+
+USB Usb; // Create an UHS2 interface object
+UHS2_GPIO GPIO(&Usb); // Create a GPIO object
+
+void setup()
+{
```

Comment:
Same here, cuddle all braces. This isn't 1979 C.

Reply:
Fixed in commit 5046bc8. To cuddle or not to cuddle isn't really a matter age but a coding style decision. Many projects I work on use uncuddled braces (and is my preference) but I did try to comply with the coding style I inferred from other software modules in this project. The example was based on the MIDI examples which do not cuddle. I did try! Anyway - sorted now and I will try to do better in the future :-).

Reply:
Not a problem :-)
Tab spacing has always been an argument, which I won't hash out here, but K&R were right on the 8 spaces, and besides, that's the default on any computer's text output and the old typewriters and printers, etc. That plus years of coding Linux Kernel have paid off stylistically.
You get a cookie for at least trying, and a beer for the patch as a whole.